pages.fr/common/git-reset.md

```diff
@@ -0,0 +1,36 @@
+# git reset
+
+> Enlève des commits ou des changements en réinitialisant la tête git à l'état spécifié
+> Si un chemin est passé, git reset fonctionne comme «unstage»;
+> Si un hash de commit est passé, git reset annule les commits jusqu'à ce dernier.
```

Comment:
I don't speak the language but I'm not sure about the `<enter>` after the `;` because is not a period `.`.
Is a French rule?
```suggestion
> Si un chemin est passé, git reset fonctionne comme «unstage»; si un hash de commit est passé, git reset annule les commits jusqu'à ce dernier.
```

Reply:
It's a French rule that you can use `;` for enumerations but in retrospective this is not really the case here. I'll use a period instead.